test/src/server/routes/collection.js

```diff
@@ -0,0 +1,221 @@
+import {createAuthor, createEditor, truncateEntities} from '../../../test-helpers/create-entities';
+
+import app from '../../../../src/server/app';
+import assertArrays from 'chai-arrays';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import orm from '../../../bookbrainz-data';
+
+
+chai.use(chaiHttp);
+chai.use(assertArrays);
+const {expect} = chai;
+const {UserCollection, UserCollectionItem} = orm;
+
+
+describe('POST /collection/create', () => {
+	let agent;
+	let collectionOwner;
+	before(async () => {
+		try {
+			collectionOwner = await createEditor(123456);
+		}
+		catch (error) {
+			// console.log(error);
+		}
+		// The `agent` now has the sessionid cookie saved, and will send it
+		// back to the server in the next request:
+		agent = await chai.request.agent(app);
+		await agent.get('/cb');
+	});
+	after((done) => {
+		// Clear DB tables then close superagent server
+		truncateEntities().then(() => {
+			agent.close(done);
+		});
+	});
+
+	it('should correctly create collection and return with status code 200 for correct data', async () => {
+		const data = {
+			description: 'some description',
+			entityType: 'Author',
+			name: 'collectionName',
+			privacy: 'public'
+		};
+		const res = await agent.post('/collection/create/handler').send(data);
+		const collection = await new UserCollection({id: res.body.id}).fetch();
+		expect(collection.get('id')).to.equal(res.body.id);
+		expect(collection.get('ownerId')).to.equal(collectionOwner.get('id'));
```

Comment:
Let's take the time to check the other attributes here (description, name, privacy, entity type).
We won't have to do it in other tests, but it doesn't cost us anything to be sure everything works as expected.